src/utils.js

```diff
@@ -178,6 +181,37 @@ export function removeQueryParam(queryString, param) {
 }
 
 
+/**
+ * Asserts that a given url is an absolute HTTP or HTTPS URL.
+ * @param {?string} urlString
+ * @return {?string}
+ */
+export function assertAbsoluteHttpOrHttpsUrl(urlString) {
+  if (!HTTP_S_ONLY_RE.test(urlString)) {
+    throw new Error('must be http(s)');
+  }
+  return urlString;
+}
+
+
+/**
+ * Asserts that a given url is not an obvious script URL. This is not intended
+ * as a complete verification. The complete verification is left up to the
+ * host before `accept()` method is called.
+ * @param {?string} urlString
+ * @return {?string}
+ */
+export function assertObviousUnsafeUrl(urlString) {
+  if (urlString) {
+    const protocol = parseUrl(urlString).protocol;
+    if (protocol.indexOf('script') != -1) {
+      throw new Error('unsafe');
```

Comment:
I'd probably include the URL in the message.

Reply:
Done